src/ripple/unl/tests/Network_test.cpp

```diff
@@ -57,7 +58,7 @@ class Net_test : public beast::unit_test::suite
         send (Net& net, Peer& from, Message&& m)
         {
             net.send (from, *this,
-                std::forward<Message>(m));
+                [&, m]() { receive(net, from, m); });
```

Comment:
Yep, this is less efficient than it could be. And the performance matters in this function.